.cursor/skills/dev-helper/phases/04-reproduce.md

```
@@ -0,0 +1,188 @@
+URL=$(.cursor/skills/dev-helper/scripts/state-cli.sh field ${TICKET_KEY} '.reproduce.consoleUrl // empty' 2>/dev/null)
+```
+
+Common URLs:
+- Local: `http://localhost:9000`
+- Cluster: `https://console-openshift-console.apps.<cluster>/`
+
+### 4.2 Verify cluster connectivity
+
+Navigate to the console and confirm it loads:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)